### ISSUE TITLE
Commands do not work

### ISSUE THREAD

**Issue by arpan98** [Terasology/Hunger#5]
All the commands throw a `NullPointerException`. This is the log for `hungerCheck`.

```
11:06:01.275 [main] ERROR o.t.logic.console.ConsoleImpl - An error occurred while executing a command
org.terasology.logic.console.commandSystem.exceptions.CommandExecutionException: java.lang.NullPointerException
	at org.terasology.logic.console.commandSystem.AbstractCommand.execute(AbstractCommand.java:242)
	at org.terasology.logic.console.ConsoleImpl.execute(ConsoleImpl.java:240)
	at org.terasology.logic.console.ConsoleImpl.execute(ConsoleImpl.java:204)
	at org.terasology.logic.console.ui.ConsoleScreen.lambda$initialise$0(ConsoleScreen.java:76)
	at org.terasology.rendering.nui.widgets.UIText.onKeyEvent(UIText.java:401)
	at org.terasology.logic.console.ui.UICommandEntry.onKeyEvent(UICommandEntry.java:83)
	at org.terasology.rendering.nui.internal.NUIManagerInternal.keyEvent(NUIManagerInternal.java:612)
	at org.terasology.rendering.nui.internal.NUIManagerInternalMethodAccess.invoke(Unknown Source)
	at org.terasology.entitySystem.event.internal.EventSystemImpl$ByteCodeEventHandlerInfo.invoke(EventSystemImpl.java:506)
	at org.terasology.entitySystem.event.internal.EventSystemImpl.sendConsumableEvent(EventSystemImpl.java:279)
	at org.terasology.entitySystem.event.internal.EventSystemImpl.send(EventSystemImpl.java:258)
	at org.terasology.entitySystem.entity.internal.BaseEntityRef.send(BaseEntityRef.java:145)
	at org.terasology.input.InputSystem.sendKeyEvent(InputSystem.java:514)
	at org.terasology.input.InputSystem.processKeyboardInput(InputSystem.java:464)
	at org.terasology.input.InputSystem.update(InputSystem.java:280)
	at org.terasology.engine.modes.StateIngame.handleInput(StateIngame.java:179)
	at org.terasology.engine.subsystem.lwjgl.LwjglInput.postUpdate(LwjglInput.java:54)
	at org.terasology.engine.TerasologyEngine.mainLoop(TerasologyEngine.java:422)
	at org.terasology.engine.TerasologyEngine.run(TerasologyEngine.java:368)
	at org.terasology.engine.Terasology.main(Terasology.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: java.lang.NullPointerException: null
	at org.terasology.hunger.HungerAuthoritySystem.hungerCheck(HungerAuthoritySystem.java:159)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.terasology.logic.console.commandSystem.AbstractCommand.execute(AbstractCommand.java:237)
```

There seems to be something wrong with the `client` argument being passed. I put a `@Sender` annotation against it according to the `@Command` doc, but that did not solve the issue.

**Comment by iojw** [comment]
@Sender actually fixes the NPE for me so #3 should fix it. Maybe your game didn't build properly?

**Comment by arpan98** [comment]
Strange. That must have been it, I might have been trying @Cervator's hot-rebuild tip.
